x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/mapping.ts

```diff
@@ -50,6 +51,21 @@ export const createConditionEntry = <T extends ConditionEntryField>(
   return { field, value, type: 'match', operator: 'included' };
 };
 
+export const tagsToEffectScope = (tags: string[]): EffectScope => {
+  const policyReferenceTags = tags.filter((tag) => tag.startsWith('policy:'));
+
+  if (policyReferenceTags.some((tag) => tag === 'policy:all')) {
+    return {
+      type: 'global',
+    };
+  } else {
+    return {
+      type: 'policy',
+      policies: policyReferenceTags.map((tag) => tag.substr(7)),
```

Comment:
Suggestion: maybe  store the prefix `policy:` in a `const` and use the length of that here. Just in case the prefix changes?

Reply:
Changed